### PR TITLE
adding rlr

### PR DESCRIPTION
**Pull request by osamples**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by osamples** [comment]
@dopplershift Here is the next one - modified to reflect the same changes that the previous one (highered) had.

File: recipes/rlr/LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+The MIT License (MIT)
```
Comment (dopplershift) [inline recipes/rlr/LICENSE:1]: Where did this license come from? It's not the one in the [upstream repo](https://github.com/dedupeio/rlr/blob/master/COPYING) and it's not an Apache 2.0 license, which what the `setup.py` for this version says it is. I'd be ok with having a generic Apache 2.0 license file here.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just need to resolve the license file, otherwise this recipe is good to go. @izahn I assume you're ok being listed as a maintainer for this recipe?

**Comment by izahn** [comment]
Yes happy to be a maintainer

**Review by dopplershift — APPROVED** [review]
Ignoring windows build failure due to missing dependency on noarch.

**Pull request merged**